pkg/storage/store_create_replica.go

```diff
@@ -132,27 +121,21 @@ func (s *Store) tryGetOrCreateReplica(
 		// If this is intended for replicaID 0 then it's either a preemptive
 		// snapshot or a split/merge lock in which case we'll let it go through.
 		if replicaID == 0 {
+			// TODO(ajwerner): fix this case.
```

Comment:
I have this all paged out again, but is the split/merge lock the last remaining use of replicaID zero? I'd be so happy to not have that be a thing. (not in this PR)

Reply:
Even better! I removed all uses of ReplicaID 0 in this PR already, just forgot to come back to this conditional after I did that. This can be a fatal now.

Reply:
❤️